### ISSUE TITLE
get_risktable() function breaks in case of "(missing)" data

### ISSUE THREAD

**Issue by lesniewa** [openpharma/visR#190]
#### Issue description
I have missing values in the dataset, which are being replaced with the "(missing)" label. 
```
(missing)         F         M 
        1     21099     16056 
```
I try to run `estimate_KM()` and then `get_risktable()` on this dataset. The `visR` plot is being rendered correctly:
![image](https://user-images.githubusercontent.com/84186593/119649563-10cdfd00-be23-11eb-9fbc-cdff88801595.png)

However, the risk table contains only the first 3 rows:
![image](https://user-images.githubusercontent.com/84186593/119649726-3e1aab00-be23-11eb-9cc5-43d62518bacd.png)

#### Steps to reproduce the issue
```R
demog_suvival_object <- g_demographics_survival %>%
  visR::estimate_KM(strata = "gender", CNSR = "hasdeathevent", AVAL = "followuptime")
gg <- visR::visr(demog_suvival_object)
gg

visR::get_risktable(demog_suvival_object)
```

#### What's the expected result?
The risk table containing all points from the graph. (0,2,4,6,8,10,12,14,16,18,20,22,24)

#### What's the actual result?
The risk table containing only 3 points from the graph. (0,2,4)

**Comment by SHAESEN2** [comment]
This is because we are using a cutoff inside the function called, min_at_risk. 

```
library(visR)

#' @param min_at_risk \code{numeric} The cutoff for number of participants at risk to display. This minimum is applied across strata. 
#'   Default is 0.

tte <- adtte
tte$SEX <- factor(tte$SEX, levels = c("F", "M", "Missing"))
tte[tte$AVAL < 3, "SEX"] <- "Missing"


survfit_obj <- estimate_KM(data = tte, strata = "SEX")

get_risktable(survfit_obj)
get_risktable(survfit_obj, min_at_risk = 50)
```

It will cut off the risktable at the earliest time min_at_risk is achieved. In your case, this defaults to 0 which is encountered the first time in the Missing group at time 4. I agree the behaviour is weird for this extreme case. I'll make another proposal.

**Comment by SHAESEN2** [comment]
Closed with #194 We removed the min_at_risk argument. In addition we ensured proper alignment of the risktable with the plot.
